diem-unified-charter.md

```diff
@@ -8,16 +8,30 @@ The DIEM WG seeks to define a standard architecture, set of emblem types, and pr
 methods that will provide a cohesive approach to creating, displaying, and
 interpreting emblems across applications and platforms.
 
+# Archtectural Consideration
+
+"To bear an emblem" means to use or adopt a symbol, figure, or device as an identifying mark. 
+The entity that bears the emblem is respectively the bearer or emblem holder. This can be a separate entity
+from the emblem issuer.
+
+To "verify an emblem" means to confirm the authenticity or legitimacy of a particular symbol or design, often by checking its details against a known standard or reference point. 
+Emblems may be observed by verifiers without the knowledge of the bearer displaying the emblem, or may be presented to a specific verifier upon request.
+However, cryptographic verification can be optional as long is the emblem can be correctly interpreted.
+Therefore, to be effective, the semantics of an emblem must be well known, easily recognizable, and distinguishable from other emblems.
+
 There are three dimensions to digital emblems: Asset Type, Emblem Type, and 
 Validator Relationship. Assets bound to digital emblems can be analog (e.g., a vehicle) or
-digital (e.g., a web server). They
-differ in the initial interaction between the asset and the validator. Digital
-Emblems can fall into multiple types (e.g., self-signed, attested, etc.) that will
+digital (e.g., a web page), however, there need to be a digital representation that is used
+for validation using digital communication.
+
+Digital emblems can fall into multiple types (e.g., self-signed, attested, etc.) that will
 affect how a validator interprets and trusts the assertions made within the emblem.
 The relationship between the asset and the validator will influence requirements on how
```

Comment:
```suggestion
The communication between the asset and the validator will influence requirements on how
```
Is it better/correct to use communication here? This is also one of Roman's comments.